### DISCUSSION TITLE
Реактивные формы ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#66] [Announcements]
# Реактивные формы ⚡️ Angular с примерами кода

Реактивные формы (Angular reactive forms) построены на основе механизма, использующего реактивный подход к программированию

https://angdev.ru/archive/angular9/angular-reactive-forms/

**Comment by antonfiluta** [comment]
эта документация на русском просто услада для глаз
